doc/world_format.md

```diff
@@ -516,6 +508,14 @@ Before map format version 29:
         * `u16` `name_len`
         * `u8[name_len]` `name`
 
+Since map format version 25:
+    * `u8` length of the data of a single timer (always 2+4+4=10)
```

Comment:
it's not totally clear that this still belongs to the "node timers" part

Reply:
Fixed.